python/src/spark_rapids_ml/tuning.py

```diff
@@ -134,3 +146,32 @@ def singePassTask(
             )
 
         return self._copyValues(model)
+
+    @staticmethod
+    def _is_python_params_instance(metadata: Dict[str, Any]) -> bool:
+        # If it's not python module, pyspark will load spark_rapids_ml.tuning.CrossValidator
+        # from JVM package. So we need to hack here
+        return metadata["class"].startswith(("pyspark.ml.", "spark_rapids_ml."))
+
+    @classmethod
+    def load(cls, path: str) -> "CrossValidator":
+        orig_is_python_params_instance = DefaultParamsReader.isPythonParamsInstance
+        try:
+            # Replace isPythonParamsInstance
+            setattr(
```

Comment:
Can we do this replacement when this package is imported?  And just have it apply to all spark_rapids_ml classes?

Reply:
Hmm, I really don't want to break the pyspark things, although we know our changing by replacing the attribute of DefaultParamsReader may not have any impaction on pyspark.